Ical.Net/DataTypes/Attendee.cs

```diff
@@ -276,10 +276,10 @@ protected bool Equals(Attendee other) => Equals(SentBy, other.SentBy)
                                              && Rsvp == other.Rsvp
                                              && Equals(Value, other.Value)
                                              && Members.SequenceEqual(other.Members)
-                                             && DelegatedTo.SequenceEqual(other.DelegatedTo)
-                                             && DelegatedFrom.SequenceEqual(other.DelegatedFrom);
+                                             && (DelegatedTo?.SequenceEqual(other.DelegatedTo ?? Enumerable.Empty<string>()) ?? other.DelegatedTo == null)
```

Comment:
This comparison is not commutative this way. I.e. if one is null and the other is empty, then the comparison is not symmetrical. Not sure though, whether an empty sequence should equal null. Usually they're considered different.